hera_sim/tests/test_simulator.py

```diff
@@ -115,11 +115,10 @@ def test_wrong_func():
     sim.add_eor("noiselike_EOR")  # wrong function name
 
 
-@raises(TypeError)
+@raises(AssertionError)
 def test_wrong_arguments():
     sim = create_sim()
-    sim.add_foregrounds(diffuse_foreground, Tsky_mdl=HERA_Tsky_mdl['xx'])
```

Comment:
this test should actually fail with the original TypeError if my other comments on this function are incorporated.